<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>362</width>
    <height>463</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
	margin: 0;
	font-size: 10.1pt;
}
QComboBox{
 	padding: 2px 6px;
}
QPushButton{
  text-align: center;
  display: inline-block;
  padding: 2px 0;
  border: 1px solid #99a;
  border-bottom: 2px solid #99a;
  letter-spacing: 1px;
  overflow:hidden;
  margin: 0 10px;
  background-color: #fafaff;
  border-radius: 2px;
}
QPushButton:hover {
    background-color: #446;
    color: white;
	border: 1px solid #fff;
}
QPushButton::disabled {
    background-color: #ccc;
    color: #777;
	border: 1px solid #999;
}
</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="4" column="0" colspan="2">
    <widget class="QPushButton" name="btnSubmit">
     <property name="maximumSize">
      <size>
       <width>382</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Last opp</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QListWidget" name="selectedItemsWidget">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::NoSelection</enum>
     </property>
     <property name="isWrapping" stdset="0">
      <bool>true</bool>
     </property>
     <property name="viewMode">
      <enum>QListView::ListMode</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QListWidget" name="listWidget">
     <property name="sortingEnabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Velg mappe(r) å laste opp&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Prosjektene er sortert etter opprettet dato med de nyeste prosjektene øverst. Har du endret status på prosjekt eller bestilt ledning vil dette lastes opp når du trykker &quot;Last opp&quot;</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
